tests/engine/state_test.py

```diff
@@ -0,0 +1,12 @@
+from scripts.engine import state
+from scripts.engine.core.constants import GameState
+
+
+class TestState:
+
+    def test_set_state(self):
```

Comment:
Very simple test. Personally I don't think getter and setters need much test coverage but they are a good practice.